repose-aggregator/components/filters/versioning-filter/src/test/java/org/openrepose/filters/versioning/util/ContentTransformerTest.java

```diff
@@ -19,39 +19,55 @@
  */
 package org.openrepose.filters.versioning.util;
 
-
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.io.IOUtils;
 import org.custommonkey.xmlunit.Diff;
-import org.junit.Before;
 import org.junit.Test;
-import org.junit.experimental.runners.Enclosed;
-import org.junit.runner.RunWith;
 import org.openrepose.commons.utils.http.media.MediaType;
 import org.openrepose.commons.utils.http.media.MimeType;
 import org.openrepose.commons.utils.transform.Transform;
 import org.openrepose.commons.utils.transform.jaxb.StreamToJaxbTransform;
+import org.openrepose.filters.versioning.config.JsonFormat;
 import org.openrepose.filters.versioning.schema.VersionChoice;
 import org.openrepose.filters.versioning.schema.VersionChoiceList;
 
 import javax.xml.bind.JAXBContext;
 import javax.xml.bind.JAXBElement;
 import java.io.ByteArrayOutputStream;
 import java.io.File;
+import java.io.IOException;
 import java.io.InputStream;
+import java.io.UncheckedIOException;
+import java.net.URISyntaxException;
 import java.util.Map;
 
-import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
 
-/**
- * @author jhopper
- */
-@RunWith(Enclosed.class)
+// suppress unchecked assignment warnings in the static initializer
+@SuppressWarnings("unchecked")
 public class ContentTransformerTest {
 
+    private static final String EXAMPLES_DIR = "/META-INF/schema/examples/";
+    private static final String XML_VERSION = EXAMPLES_DIR + "xml/version.xml";
+    private static final String XML_VERSIONS = EXAMPLES_DIR + "xml/versions.xml";
+    private static final String JSON_VERSION = EXAMPLES_DIR + "json/version.json";
+    private static final String JSON_VERSIONS = EXAMPLES_DIR + "json/versions.json";
+    private static final String JSON_VERSION_IDENTITY = EXAMPLES_DIR + "json/version-identity.json";
+    private static final String JSON_VERSIONS_IDENTITY = EXAMPLES_DIR + "json/versions-identity.json";
+    private static final String XML_CHOICES = EXAMPLES_DIR + "xml/choices.xml";
+    private static final String JSON_CHOICES = EXAMPLES_DIR + "json/choices.json";
+
     private static final Transform<InputStream, JAXBElement<?>> xmlTransformer;
 
+    private ObjectMapper mapper = new ObjectMapper();
+    private String versionXmlFileReader = getFileAsString(XML_VERSION);
+    private String versionsXmlFileReader = getFileAsString(XML_VERSIONS);
+
     static {
```

Comment:
Dang it.  I also moved some other stuff around because I thought this comment was on another line.  In any case, I did get rid of the static initializer.